podman/domain/config.py

```diff
@@ -64,15 +69,22 @@ def __init__(self, path: Optional[str] = None):
 
         if path is None:
             home = Path(xdg.BaseDirectory.xdg_config_home)
-            self.path = home / "containers" / "containers.conf"
+            self.path = home / "containers" / "podman-connections.json"
+            old_toml_file = home / "containers" / "containers.conf"
```

Comment:
Does this work on windows? The windows containers.conf path differ but I am not sure if this repo cares about windows support at all?

Reply:
I am not sure about our windows story, @jwhonce do you have any idea?

Reply:
It used to work but did not use the windows registry. I no longer have a windows license to spin on a VM for testing.

Reply:
well works yeah sure it can read these paths but these are not the paths podman actually uses on windows